### ISSUE TITLE
Stellar parameters, including v sin i

### ISSUE THREAD

**Issue by jradavenport** [jradavenport/spectra#65]
It would be nice to have a secondary package to measure basic stellar parameters, including [v sin i](https://ui.adsabs.harvard.edu/#abs/2011A%26A...531A.143D)

Parameters could include:
- T_eff
- Spectral type (like the Hammer)
- RV
- v sin i
- equivalent widths of a few key lines

Once PyDIS is ready for it's next release (is tested, ships w/ sample data, etc), this would make a nice student project to build useful features.

These functions could (and should?) be more general than for PyDIS, and could be included in astropy
